### PR TITLE
Fix warnings for OCFileListFragment

### PR DESCRIPTION
**Pull request by alperozturk96**
- [x] Tests written, or not not needed

### CODE REVIEW

File: app/src/main/java/com/owncloud/android/ui/fragment/OCFileListFragment.java [hunk 1]
```diff
@@ -1786,38 +1734,8 @@ private void encryptFolder(OCFile folder,
         }
     }
 
-    @Subscribe(threadMode = ThreadMode.BACKGROUND)
-    public void onMessageEvent(FileLockEvent event) {
```
Comment (alperozturk96) [inline app/src/main/java/com/owncloud/android/ui/fragment/OCFileListFragment.java:1790]: Unused function

Comment (AndyScherzinger) [reply app/src/main/java/com/owncloud/android/ui/fragment/OCFileListFragment.java:1790]: Like mentioned in the other PR, I expect the event bus to make use of them, so this should actually be a false-positive, best to check with @tobiasKaminsky

Comment (tobiasKaminsky) [reply app/src/main/java/com/owncloud/android/ui/fragment/OCFileListFragment.java:1790]: Yeah, this is still used.
You can check where FileLockEvent is being used:
Within FileOperationsHelper there is: EventBus.getDefault().post(new FileLockEvent(file.getRemotePath(), shouldBeLocked));

This is to send the event, and the other function receives it.

In long term I would like to get rid of it…

**Review by tobiasKaminsky — CHANGES_REQUESTED** [review]
Re-add event bus function.

File: app/src/main/java/com/owncloud/android/ui/fragment/OCFileListFragment.java [hunk 2]
```diff
@@ -421,8 +430,7 @@ public void onActivityCreated(Bundle savedInstanceState) {
         setTitle();
 
         FragmentActivity fragmentActivity;
-        if ((fragmentActivity = getActivity()) != null && fragmentActivity instanceof FileDisplayActivity) {
-            FileDisplayActivity fileDisplayActivity = (FileDisplayActivity) fragmentActivity;
+        if ((fragmentActivity = getActivity()) != null && fragmentActivity instanceof FileDisplayActivity fileDisplayActivity) {
```
Comment (ZetaTom) [inline app/src/main/java/com/owncloud/android/ui/fragment/OCFileListFragment.java:433]:
```suggestion
        if (getActivity() instanceof FileDisplayActivity fileDisplayActivity) {
```

File: app/src/main/java/com/owncloud/android/ui/fragment/OCFileListFragment.java [hunk 3]
```diff
@@ -1786,6 +1731,7 @@ private void encryptFolder(OCFile folder,
         }
     }
 
+    // Do not delete this function
```
[the remark this answers is not in the document]
Comment (alperozturk96) [reply app/src/main/java/com/owncloud/android/ui/fragment/OCFileListFragment.java:1733]: IDE will always show as unused function better to keep that comment

**Comment by alperozturk96** [comment]
Out of date due to conflicts

**Pull request closed without merging**